### PR TITLE
[example] Add Facebook Chat Plugin example

### PR DESCRIPTION
**Pull request by ruffac**
This PR adds an FB Chat plugin example

## Documentation / Examples

- [x] Make sure the linting passes by running `yarn lint`

### CODE REVIEW

**Comment by aecorredor** [comment]
@ruffac I just used this example to integrate it into my Next.js commerce website and it worked, but I noticed the CSS is broken after clicking on the chat icon (when you open it up). A bunch of weird elements come into view (a couple of small white dots on the bottom right and weird padding around the chat window with a transparent background). Have you seen this before?

File: examples/with-facebook-chat-plugin/.env.local.example [hunk 1]
```diff
@@ -0,0 +1 @@
+FACEBOOK_PAGE_ID=
```
Comment (balazsorban44) [inline examples/with-facebook-chat-plugin/.env.local.example:1]:
```suggestion
NEXT_PUBLIC_FACEBOOK_PAGE_ID=
```

See: https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

File: examples/with-facebook-chat-plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+pnpm create next-app -- --example with-facebook-chat-plugin with-facebook-chat-plugin-app
+```
+
+Next, copy the `.env.local.example` file in this directory to `.env.local` (which will be ignored by Git):
+
+```bash
+cp .env.local.example .env.local
+```
+
+Set the `FACEBOOK_PAGE_ID` variable in `.env.local` to match your Facebook page's ID.
```
Comment (balazsorban44) [inline examples/with-facebook-chat-plugin/README.md:23]:
```suggestion
Set the `NEXT_PUBLIC_FACEBOOK_PAGE_ID` variable in `.env.local` to match your Facebook page's ID.
```

File: examples/with-facebook-chat-plugin/package.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "next": "latest",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  }
```
Comment (balazsorban44) [inline examples/with-facebook-chat-plugin/package.json:12]:
```suggestion
    "react": "^18.1.0",
    "react-dom": "^18.1.0"
  },
  "devDependencies": {
    "@types/node": "^17.0.42",
    "@types/react": "^18.0.12",
    "typescript": "^4.7.3"
  }
```

File: examples/with-facebook-chat-plugin/pages/_app.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import type { AppProps } from 'next/app'
+import Script from 'next/script'
+
+const FACEBOOK_PAGE_ID = process.env.FACEBOOK_PAGE_ID
```
Comment (balazsorban44) [inline examples/with-facebook-chat-plugin/pages/_app.tsx:4]:
```suggestion
const FACEBOOK_PAGE_ID = process.env.NEXT_PUBLIC_FACEBOOK_PAGE_ID
```

File: examples/with-facebook-chat-plugin/pages/index.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { NextPage } from 'next/types'
+
+const Home: NextPage = () => {
+  return (
+    <div>
+      <h1>
+        Go to `pages/_app.tsx` to add your Facebook page ID in order for the
+        chat plugin to show.
```
Comment (balazsorban44) [inline examples/with-facebook-chat-plugin/pages/index.tsx:8]:
```suggestion
        Add `NEXT_PUBLIC_FACEBOOK_PAGE_ID` to `.env.local` in order for the
        chat plugin to show.
```

**Comment by leerob** [comment]
Is this ready to be reviewed?

**Comment by leerob** [comment]
Closing as stale, but happy to review a new PR if you are still interested in contributing here!

**Pull request closed without merging**